src/netappfiles-preview/azext_netappfiles_preview/custom.py

```diff
@@ -28,125 +29,136 @@ def generate_tags(tag):
+
+#     return client.create_or_update(body, resource_group_name, account_name, pool_name, volume_name)
+
+
+# region volume
+class VolumeCreate(_VolumeCreate):
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        from azure.cli.core.aaz import AAZStrArg, AAZIntArgFormat
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        args_schema.vnet = AAZStrArg(
+            options=["--vnet"],
+            arg_group="Properties",
+            help="Name or Resource ID of the vnet. If you want to use a vnet in other resource group or subscription, please provide the Resource ID instead of the name of the vnet.",
+            required=True,
+        )
+
+        args_schema.usage_threshold._fmt = AAZIntArgFormat(
+            maximum=500,
+            minimum=100,
+        )
+
+        return args_schema
+
+    def pre_operations(self):
+        args = self.ctx.args
+        # RP expects bytes but CLI allows integer TiBs for ease of use
+        logger.debug("ANF-Extension log: usage_threshold: %s", args.usage_threshold)
+        if args.usage_threshold is not None:
+            gib_scale = 1024 * 1024 * 1024
+            args.usage_threshold = int(args.usage_threshold.to_serialized_data()) * gib_scale
+
+        # default the resource group of the subnet to the volume's rg unless the subnet is specified by id
+        subnet_rg = args.resource_group
+        subs_id = self.ctx.subscription_id
+        vnetArg = args.vnet.to_serialized_data()
+        # determine vnet - supplied value can be name or ARM resource Id
+        if is_valid_resource_id(vnetArg):
+            resource_parts = parse_resource_id(vnetArg)
+            vnetArg = resource_parts['resource_name']
+            subnet_rg = resource_parts['resource_group']
+
+        # determine subnet - supplied value can be name or ARM resource Id
+        if is_valid_resource_id(args.subnet_id.to_serialized_data()):
+            resource_parts = parse_resource_id(args.subnet_id.to_serialized_data())
+            subnet = resource_parts['resource_name']
+            subnet_rg = resource_parts['resource_group']
+
+        args.subnet_id = f"/subscriptions/{subs_id}/resourceGroups/{subnet_rg}/providers/Microsoft.Network/virtualNetworks/{vnetArg}/subnets/{subnet}"
+
+        # if NFSv4 is specified then the export policy must reflect this
+        # the RP ordinarily only creates a default setting NFSv3.
+        logger.debug("ANF-Extension log: ProtocolTypes rules len:%s", len(args.protocol_types))
+
+        for protocl in args.protocol_types:
+            logger.debug("ANF-Extension log: ProtocolType: %s", protocl)
+
+        logger.debug("ANF-Extension log: exportPolicy rules len:%s", len(args.rules))
+
+        for rule in args.rules:
+            logger.debug("ANF-Extension log: rule: %s", rule)
+
+        if (args.protocol_types is not None and any(x in ['NFSv3', 'NFSv4.1'] for x in args.protocol_types) and len(args.rules) == 0)\
```

Comment:
We use `has_value` to verify an argument. This function should be imported from `from azure.cli.core.aaz import has_value`
```suggestion
        if (has_value(args.protocol_types) and any(x in ['NFSv3', 'NFSv4.1'] for x in args.protocol_types) and len(args.rules) == 0)\
```